deployments/helm-chart/values.yaml

```diff
@@ -221,17 +221,22 @@ controller:
   ## The number of replicas of the Ingress Controller deployment.
   replicaCount: 1
 
-  ## A class of the Ingress Controller.
+  # Configures the ingress class the Ingress Controller uses.
+  ingressClass:
```

Comment:
This might be a breaking change? Since we're changing `ingressClass` from a string to an object?

Reply:
Yes, we will require the users to use the correct version of the helm chart and update their helm value yaml file before installing a new release, otherwise the helm schema will reject it. Is it what you meant?

Reply:
what I mean is that we probably have to bump the helm chart to `1.0.0`

Reply:
We decided to bump the helm chart to 1.0.0